### PR TITLE
Update matcher to take list of arguments; alias it as #strip_attributes

### PR DESCRIPTION
**Pull request by experimatt**
@rmm5t here's a PR for the issue I created: https://github.com/rmm5t/strip_attributes/issues/51

I tried to match your style as much as possible, but please let me know if you'd like me to make any changes.

### CODE REVIEW

File: lib/strip_attributes/matchers.rb [hunk 1]
```diff
@@ -6,26 +6,35 @@ module Matchers
...
       def matches?(subject)
-        subject.send("#{@attribute}=", " string ")
-        subject.valid?
-        subject.send(@attribute) == "string" and collapse_spaces?(subject)
+        @attributes.map do |attribute|
+          @attribute = attribute
+          subject.send("#{@attribute}=", " string ")
+          subject.valid?
+          subject.send(@attribute) == "string" and collapse_spaces?(subject)
+        end.all?
```
Comment (rmm5t) [inline lib/strip_attributes/matchers.rb]: [PLEASE CHANGE] no need to pass over the array twice:

```ruby
        @attributes.all? do |attribute|
          @attribute = attribute
          subject.send("#{@attribute}=", " string ")
          subject.valid?
          subject.send(@attribute) == "string" and collapse_spaces?(subject)
        end
```

Comment (experimatt) [reply lib/strip_attributes/matchers.rb]: Good catch! Updated, and I also removed the extra unnecessary splats (`*`)

Comment (experimatt) [reply lib/strip_attributes/matchers.rb]: Would you like me to squash the commits into one, or is there an automatic way github can do that with the merge (I'm used to gitlab)?

**Comment by rmm5t** [comment]
Squashed and merged. Thank you! 🍻

**Comment by rmm5t** [comment]
v.1.10.0 release with these changes!
https://rubygems.org/gems/strip_attributes

**Pull request merged**